master/requirements-dev.txt

```diff
@@ -0,0 +1,11 @@
+twisted == 17.5.0
+Jinja2 == 2.8
+sqlalchemy >= 0.6, <= 0.7.9
+sqlalchemy-migrate == 0.7.2
+python-dateutil == 1.5
+autobahn == 0.11.0
+python-ldap
+mysql-python == 1.2.5
+pymysql == 0.7.1
+PyJWT == 1.4.0
+psutil == 4.3.0
```

Comment:
@campos-ddc in https://github.com/Unity-Technologies/katana/pull/296 mentioned:
> Aren't any of these declared in setup.py? IMO requirements-dev should only contain things that are required for development but not every day use (the every day stuff should be in setup.py)

So let's continue this thread here.

Reply:
Dockerfile will be used in the developement, won’t it be? Indeed, it’s almost the same as in setup.py and in the current approach we are avoiding any package installation.

Reply:
Not sure if I got what you mean, this is what I would expect:

* If you want to develop new code for Katana, you have to run `setup.py` and `pip install` requirements-dev
* If you just want to start a server, running `setup.py` should be enough.

(doesn't matter if you are using a Dockerfile or not ; `setup.py` will usually be ran when doing a `pip install -e katana/`)